casper/src/main/resources/PoS.rhox

```diff
@@ -240,6 +253,10 @@ new  PoS,
         }
       } |
 
+      contract fst(@(first, _), resultCh) = {
+        resultCh!(first)
```

Comment:
Can we use an in-line match statement instead?

Reply:
not really. I need a lambda contract to pass to the ListOps.map